infrastructure/tools/s3-cache/s3-cache-lambda.ts

```diff
@@ -0,0 +1,64 @@
+import { S3, CodePipeline } from 'aws-sdk'
+
+const fileToModifyCache = [ 'index.html' ]
+const s3 = new S3()
+const codepipeline = new CodePipeline()
+
+export function handler(event: string, context: any){
+    const jobId = event["CodePipeline.job"].id
+    const putJobFailure = function(message: any) {
+        const params = {
+            jobId,
+            failureDetails: {
+                message: JSON.stringify(message),
+                type: 'JobFailed',
+                externalExecutionId: context.awsRequestId,
+            },
+        }
+        codepipeline.putJobFailureResult(params, (err, data) => {
+            context.fail(message)
+        })
+    }
+
+    const putJobSuccess = function(message: any) {
+        const params = {
+            jobId
+        }
+        codepipeline.putJobSuccessResult(params, (err, data) => {
+            if(err) {
+                context.fail(err)
+            } else {
+                context.succeed(message)
+            }
+        })
+    }
+    try {
+        const userParams = JSON.parse(event["CodePipeline.job"].data.actionConfiguration.configuration.UserParameters)
+        console.log(userParams)
```

Comment:
What is the reason for returning `userParams` in the console?

Reply:
So we are actually using this for logging ironically haha. Not common in web apps, but common with lambdas so we can see what the inputs are and debug executions